_layouts/base.html

```diff
@@ -3,17 +3,16 @@
 <!--
     ============================================================================
 
-    Hey! If you're viewing this, you should probably come work on our Technology
-    & Innovation team. We're always looking for a few great designers,
-    developers, data scientists, and network, infrastructure, privacy and
-    security pros. Keep an eye on our job opportunities at:
-    http://www.consumerfinance.gov/jobs/
+    Hey! If you're viewing this, you should probably come work on our
+    Technology & Innovation team. We're always looking for a few great
+    designers; developers; data scientists; and network, infrastructure,
```

Comment:
It's true that the semi-colon rule @Scotchester referenced is to be used when the items being put in series contain commas, as happens with city-state listings or long phrases. For this sentence, it makes the whole thing bonkers and unreadable. This wouldn't get past a professional copyeditor in either edit.

Could we split it up into a unordered list, if that's what we're actually trying to convey?